src/status_im/contexts/wallet/create_account/new_keypair/backup_recovery_phrase/view.cljs

```diff
@@ -44,8 +45,14 @@
                                         :2 false
                                         :3 false})
         revealed?                     (reagent/atom false)
-        {:keys [customization-color]} (rf/sub [:profile/profile])]
+        {:keys [customization-color]} (rf/sub [:profile/profile])
+        secret-phrase                 (reagent/atom [])
+        random-phrase                 (reagent/atom [])]
     (fn []
+      (rn/use-effect
+       (fn []
+         (native-module/get-random-mnemonic #(reset! secret-phrase (utils/string-to-vector %)))
```

Comment:
it's not immediately clear why we need this? Can you please explain why we have this here and how it works?

Reply:
@J-Son89 to generate the random mnemonic (secret phrase) for the keypair

Reply:
ah okay, yeah should be clear enough. I was getting confused that there was some clean-up function here, but seems I misread and all good. Thanks Omar!